rails_programming/forms_and_authentication/project_forms.md

```diff
@@ -43,7 +43,7 @@ The first form you build will be mostly HTML (remember that stuff at all?).  Bui
 
    ~~~bash
    Started POST "/users" for 127.0.0.1 at 2013-12-12 13:04:19 -0800
-   Processing by UsersController#create as HTML
+   Processing by UsersController#create as TURBO_STREAM
```

Comment:
A lot of steps don't work like this anymore:

1. OK
2. OK
3. You don't get an error, turbo uses the authenticity token from the meta tag and submit that if there is not authenticity token in the form. So only if we opt out from turbo will you see that error.  Instead nothing happens and you get back a `204 No Content`  
4. So if you do step 4 nothing changes
5. We still get a 204 No content response
From 6 on, it works again as expected

So either we need to opt out the form of turbo to illustrate the concepts or to shorten those steps.